### PR TITLE
updated repo_auth test on version 6.6

### PR DESCRIPTION
**Pull request by vijay8451**
- updated repo_auth test on version 6.6
- Currently test is failing with below error:
```
AssertionError: 'true' not found in 'Loading production environment (Rails 5.2.1)\r\nSwitch to inspect mode.\r\nrepo = Katello::Repository.find_by_id(632); repo.upstream_username = "rTtest123"; repo.save!(validate: false)\r\n\x1b[1mTraceback\x1b[m (most recent call last):\r\n        2: from lib/tasks/console.rake:5:in `block in <top (required)>\'\r\n        1: from (irb):1\r\n\x1b[1mNoMethodError (\x1b[4mundefined method `upstream_username=\' for #<Katello::Repository:0x000000001039bb78>\x1b[0;1m)\x1b[m\r\n\x1b[1mDid you mean?  upstream_username\x1b[m'
```

### CODE REVIEW

File: tests/upgrades/test_repository.py [hunk 1]
```diff
@@ -33,75 +33,6 @@
...
-
-        :expectedresults:
-            1. Upstream username should be set on repository.
-
-        :BZ: 1641785
-        """
-
-        org = entities.Organization().create()
-        custom_repo = create_sync_custom_repo(org_id=org.id)
-        rake_repo = 'repo = Katello::Repository.find_by_id({0})'.format(custom_repo)
```
Comment (ntkathole) [inline tests/upgrades/test_repository.py]: @vijay8451 I see upstream username and password in UI. May be rake command changed here?

Comment (vijay8451) [reply tests/upgrades/test_repository.py]: @ntkathole yeah it's there but you can not update the standalone username or password as you have to give both in same time.
Another way do earlier in 6.4 that was `rake console` but check is made from `6.5` hence it is  not allowing now same and applicable too:
```
irb(main):002:0> repo.upstream_
repo.upstream_password  repo.upstream_username  
irb(main):002:0> repo.upstream_username = 'test123'
Traceback (most recent call last):
        2: from lib/tasks/console.rake:5:in `block in <top (required)>'
        1: from (irb):2
NoMethodError (undefined method `upstream_username=' for #<Katello::Repository:0x000000000ad6cfb8>)
Did you mean?  upstream_username
irb(main):003:0> repo.upstream_password = 'password'
Traceback (most recent call last):
        2: from lib/tasks/console.rake:5:in `block in <top (required)>'
        1: from (irb):3
NoMethodError (undefined method `upstream_password=' for #<Katello::Repository:0x000000000ad6cfb8>)
Did you mean?  upstream_password
irb(main):004:0> 

```

Comment (ntkathole) [reply tests/upgrades/test_repository.py]: @vijay8451 Could you please try with `repo.root.upstream_username = "test123"` ? This is changed via https://projects.theforeman.org/issues/24328

Comment (vijay8451) [reply tests/upgrades/test_repository.py]: @ntkathole great to see that .. I was not aware about it. let me just update the command only that all :)

**Comment by vijay8451** [comment]
@ntkathole revisit please .

**Comment by vijay8451** [comment]
```
irb(main):001:0> repo = Katello::Repository.find_by_id(2)
=> #<Katello::Repository id: 2, pulp_id: "Default_Organization-Red_Hat_Enterprise_Linux_Serv...", library_instance_id: nil, content_view_version_id: 1, relative_path: "Default_Organization/Library/content/dist/rhel/ser...", environment_id: 1, saved_checksum_type: "sha1", distribution_version: nil, distribution_arch: nil, distribution_bootable: nil, distribution_family: nil, distribution_variant: nil, distribution_uuid: nil, container_repository_name: nil, root_id: 7, remote_href: nil, publication_href: nil, version_href: nil>

irb(main):003:0> repo.root.upstream_username = 'test123123'
=> "test123123"

irb(main):005:0> repo.root.upstream_password
=> nil
irb(main):006:0> repo.root.upstream_password = 'wrwerw'
=> "wrwerw"
irb(main):007:0> repo.root.upstream_password
=> "wrwerw"
```

**Pull request merged**